### ISSUE TITLE
pip installation conflict on Xenial charms built from layer-basic

### ISSUE THREAD

**Issue by marcoceppi** [juju-solutions/layer-basic#71]
Any charm being deployed on Xenial which was built from layer-basic is failing due to resource conflict. It seems that installing 7.1.2 when 8.1.1 is on the system.

```
pkg_resources.VersionConflict: (pip 8.1.1 (/usr/lib/python3/dist-packages), Requirement.parse('pip===7.1.2'))
```

This is failing for all charms built with this layer on Xenial. It appears something /today/ was updated in Xenial to cause this. #70 attempts to address this, but the repercussion of this is pip 8.1.1 will be installed on trusty machines. This does not appear to be a problem at the moment.

**Comment by chuckbutler** [comment]
I don't think we need to gate on this then, as failing xenial builds are holding up progress on client deliverable work at the moment.

**Comment by johnsca** [comment]
I think this was resolved by #70.  Reopen if that's not the case.

**Comment by kwmonroe** [comment]
i can confirm we hit this with apache-spark on xenial (version conflict when pip-7.1.2 was in the wheelhouse).  rebuilding with the layer basic changes from #70 fixed this for us.
